lib/__tests__/click.test.ts

```diff
@@ -1,11 +1,16 @@
-import AlgoliaInsights from "../insights";
+import { getInstance } from "../../tests/utils";
 
 const credentials = {
   apiKey: "test",
   appId: "test"
 };
 
 describe("clickedObjectIDsAfterSearch", () => {
+  let AlgoliaInsights;
+  beforeEach(() => {
```

Comment:
Top level `beforeEach` does not work?

Reply:
Surprisingly I didn't know `beforeEach` can be used outside `describe`.. https://github.com/algolia/search-insights.js/pull/140/commits/7f4217b2acd67707aaced75cccc537c28dd49da0